packages/features/bookings/components/event-meta/Details.tsx

```diff
@@ -108,7 +109,11 @@ export const EventMetaBlock = ({
  * const MyCustomBlock = () => <div>Something nice</div>;
  * <EventDetails event={event} blocks={[EventDetailBlocks.LOCATION, MyCustomBlock]} />
  */
-export const EventDetails = ({ event, blocks = defaultEventDetailsBlocks }: EventDetailsProps) => {
+export const EventDetails = ({
+  event,
+  blocks = defaultEventDetailsBlocks,
+  rescheduleUid,
```

Comment:
Can't we just use 

```
const rescheduleUid = useBookerStore((state) => state.rescheduleUid);
```

In here instead of doing prop-drilling?

https://handbook.cal.com/engineering/best-practices/avoid-prop-drilling